chroma-manager.conf.template

```diff
@@ -139,6 +139,8 @@ server {
         proxy_set_header X-Forwarded-Host $host;
         proxy_set_header X-Forwarded-Server $host;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_pass_request_body off;
+        proxy_set_header Content-Length "";
```

Comment:
Tested this change doesn't cause any other issues?

Reply:
I haven't seen any issues with it. There is no need to pass the body during authorization. The docs also use this method: http://nginx.org/en/docs/http/ngx_http_auth_request_module.html

I'll follow up shortly showing that we can't access certain pages when logged out still.

Reply:
After a fresh deploy and logging in I try to access grafana:
![image](https://user-images.githubusercontent.com/1100657/72546748-53f48500-3859-11ea-803e-dc3eccf0a30b.png)

After Logging out:
![image](https://user-images.githubusercontent.com/1100657/72546800-6969af00-3859-11ea-930e-bd672580ee94.png)